recipes/mpir/meta.yaml

```diff
@@ -0,0 +1,34 @@
+{% set version = "2.7.2" %}
+
+package:
+  name: mpir
+  version: {{ version }}
+
+source:
+  fn: mpir-{{ version }}.tar.bz2
+  url: http://mpir.org/mpir-{{ version }}.tar.bz2
+
+build:
+  number: 0
```

Comment:
To make sure this doesn't build on Mac and Linux (as I am guessing it doesn't), just add the following to the `build` section.

    skip: true  # [unix]